1.Core/UIElements/UIElements.csproj

```diff
@@ -9,11 +9,12 @@
 
   <ItemGroup>
     <PackageReference Include="DotNetSeleniumExtras.WaitHelpers" Version="3.11.0" />
-    <PackageReference Include="Selenium.WebDriver" Version="4.8.0" />
+    <PackageReference Include="Selenium.WebDriver" Version="4.1.0" />
```

Comment:
Why did you downgrade this package?

Reply:
its because of the requirements for browserstack, if we use the version 4.8.0, it's gonna throw this error:
![imagen](https://user-images.githubusercontent.com/5541808/221644588-89516f6c-7f68-4b23-8245-f5a0c1f0dd55.png)

Reply:
and that's because we are using this obsolete method in the browserstackWebDriver.cs
![imagen](https://user-images.githubusercontent.com/5541808/221644991-9754fefa-06d7-4b68-b977-c8e2d317feac.png)